src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -54,35 +53,11 @@ private static void PlatformInitialize()
 
             private static void InitializeSystemContext()
             {
-                if (Environment.GetEnvironmentVariable("not_supported_MONO_MWF_USE_NEW_X11_BACKEND") != null || Environment.GetEnvironmentVariable("MONO_MWF_MAC_FORCE_X11") != null)
-                {
-                    UseX11Drawable = true;
-                }
-                else
-                {
-                    IntPtr buf = Marshal.AllocHGlobal(8192);
-                    // This is kind of a hack but gets us sysname from uname (struct utsname *name) on
-                    // linux and darwin
-                    if (uname(buf) != 0)
-                    {
-                        // WTH: We couldn't detect the OS; lets default to X11
-                        UseX11Drawable = true;
-                    }
-                    else
-                    {
-                        string os = Marshal.PtrToStringAnsi(buf);
-                        if (os == "Darwin")
-                            UseCarbonDrawable = true;
-                        else
-                            UseX11Drawable = true;
-                    }
-                    Marshal.FreeHGlobal(buf);
-                }
+                // We use X11 on all platforms except for macOS, where Carbon Drawing can be used.
+                // On macOS, users can force X11 by setting the SYSTEM_DRAWING_COMMON_FORCE_X11 flag.
+                UseX11Drawable = !RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || Environment.GetEnvironmentVariable("SYSTEM_DRAWING_COMMON_FORCE_X11") != null;
```

Comment:
Is this InitializeSystemContext function still necessary? Just wondering if it should be consolidated down to:
```C#
public static bool UseX11Drawable { get; } =
    !RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ||
    Environment.GetEnvironmentVariable("SYSTEM_DRAWING_COMMON_FORCE_X11") != null;
```
?

Reply:
Yup, fixed it.